### ISSUE TITLE
False negative on valid-expect

### ISSUE THREAD

**Issue by creage** [playwright-community/eslint-plugin-playwright#152]
We use [wait-for-expect](https://github.com/TheBrainFamily/wait-for-expect) package, which gives false negative for the `valid-expect` rule on this code

```ts
import waitForExpect from 'wait-for-expect';

waitForExpect(() => {
    ...
})
```

**Comment by mskelton** [comment]
That package is unnecessary. You can use `expect.poll` instead which is builtin to Playwright.

https://playwright.dev/docs/test-assertions#expectpoll

**Comment by creage** [comment]
@mskelton What do you mean?

The bug is still there, the rule should not fail my linting if I use that package. I'm not gonna refactor our tests just make that rule pass, right?

**Comment by mskelton** [comment]
This package is built with the assumption that you are using Playwright and Playwright Test fairly conventionally.  Lint rules that check page methods will check both `page` as well as `...Page` to account for custom page objects. Similarly with `expect` we also check `...Expect` to account for custom expect via `expect.configure`.

It's obviously a trade off, if we were stricter about only checking `page` and `expect`, we would have less false negatives for use cases like yours, but we would also have more false positives for the other instances I mentioned which are significantly more common.

I'm sorry you are experiencing this issue, but I do believe the direction we've gone with this plugin is the best for the community at large.
